python_functionality/homework.py

```diff
@@ -14,7 +14,12 @@ def task_1_fix_names_start_letter(data: DT) -> DT:
         fix_names_start_letters([{'name': 'Alex', 'age': 26}, {'name': 'denys', 'age': 89}])
         >>> [{'name': 'Alex', 'age': 26}, {'name': 'Denys', 'age': 89}]
     """
-    pass
+    return [
+        {
+            'name': d['name'].capitalize(),
+            'age': d['age']
+        } if d.get('name') else {'age': d['age']} for d in data
+    ]
```

Comment:
do not format this so

Reply:
how am i supposed to format it if length of line is more than 80 characters?

Reply:
fixed